doc/fuzzing.md

```diff
@@ -87,11 +102,33 @@ To build all fuzz targets with libFuzzer, run
 make
 ```
 
-The fuzzer needs some inputs to work on, but the inputs or seeds can be used
-interchangeably between libFuzzer and AFL.
-
 See https://llvm.org/docs/LibFuzzer.html#running on how to run the libFuzzer
 instrumented executable.
 
 Alternatively run the script in `./test/fuzz/test_runner.py` and provide it
 with the `${DIR_FUZZ_IN}` created earlier.
+
+### macOS hints for libFuzzer
+
+The default clang/llvm version supplied by Apple on MacOS does not include
+fuzzing libraries, so macOS users will need to install a full version, for
+example using `brew install llvm`.
+
+When executing ./configure you should put --disable-asm to avoid errors
+with certain assembly code from Bitcoin Core's code.
+
+You may also need to take care of giving the correct path for clang and
+clang++, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
+clang does not come first in your path.
+
+If you run into problems with "boost sleep" or some of boost's libraries
```

Comment:
What errors are you seeing with Boost? Can you post logs?

Reply:
I have not seen this particular error myself but included it for others since it seems to have been an issue here: https://github.com/bitcoin/bitcoin/issues/17657#issuecomment-562869600. So I can remove it or improve it as you describe in the comment on the flags. Or maybe @brakmic can help (sorry for including you unsolicited).

Reply:
I have seen such errors when experimenting with libFuzzer and after some search through GitHub I found a solution to a similar problem: macOS should be provided an **isysroot** variable that points to the proper SDK. However, this could maybe only be a problem when working with macOs Catalina. I've never tested it under older variants. Catalina, in many ways, is a "breaking change" regarding handling of SDK paths, include directories etc.

Reply:
Updated to use xcrun --show-sdk-path.